.github/workflows/labels.yml

```diff
@@ -3,6 +3,7 @@ on: [issues, pull_request, workflow_dispatch]
 jobs:
   sync-labels:
     runs-on: ubuntu-latest
+    if: github.repository == 'cds-snc/cra-arc-upd-tbpc'
```

Comment:
Same! If anything those are standard workflow that CDS provides and you don't want them different.